Project.toml

```diff
@@ -7,8 +7,11 @@ version = "0.1.9"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+Tulip = "6dd1b50a-3aae-11e9-10b5-ef983d2400fa"
+VegaLite = "112f6efa-9a02-5b7d-90c0-432ed331239a"
```

Comment:
These should only be test dependencies, it seems?

Reply:
It seems my Project.toml was modified and I didn't notice. Yeah, this are not supposed to be dependencies, only on testing.

Reply:
I already made a PR: https://github.com/zsteve/OptimalTransport.jl/pull/23